lazy_dataset/core.py

```diff
@@ -767,42 +767,34 @@ def batch(self, batch_size: int, drop_last: bool = False) -> 'BatchDataset':
         return BatchDataset(self, batch_size, drop_last)
 
     def batch_bucket_dynamic(
-            self, batch_size, key, max_padding_rate, total_size_threshold=None,
-            expiration=None, drop_incomplete=False, sort_by_key=False):
+            self, bucket_cls, expiration=None, drop_incomplete=False,
+            sort_key=None, reverse_sort=False, *bucket_args, **bucket_kwargs):
```

Comment:
Is there a reason to support `*bucket_args`?
I would like to drop it.
All arguments, except the first should be given with key value.